plugin/inc/views/skippedpayment.class.php

```diff
@@ -43,6 +44,7 @@ protected function getSettings(): array
 							nepla on nepla.codbenef = b.codbenef
 						LEFT JOIN {$this->table_prefix}hmarfa_facturi f on f.tip LIKE 'TF%' AND (codl = 'F' OR stare like 'V%') AND f.codbenef = b.codbenef AND f.datafac = nepla.min
 						LEFT JOIN glpi_users u on SUBSTRING(u.name FROM 1 FOR 3) = SUBSTRING(f.nrcmd FROM 1 FOR 3)
+						LEFT JOIN glpi_plugin_fields_suppliersuppliercustomfields s on s.hmarfa_code_field = e.cod
```

Comment:
Let's be consistent with the table aliases
```suggestion
						LEFT JOIN glpi_plugin_fields_suppliersuppliercustomfields scf on s.hmarfa_code_field = e.cod
```